### PR TITLE
yuzu: mention GPLv3.0 in about dialog

### LINKED ISSUE #8218: GPL version clarification
Lately you changed the project's license to GPLv3+. src/yuzu/aboutdialog.ui and the translation files still say it is GPLv2.0. Please update those references.

### PR DESCRIPTION
**Pull request by Tachi107**
Follow-up to 284934ebfdf5e530c960cf69969172ff76f40bea

Fixes #8218

Only tested with the Italian translation, but it should work fine for all the languages

image

### CODE REVIEW

**Comment by FernandoS27** [comment]
it's GPLv3+ not plain GPLv3 even though GPLv4 does not exist, we want to leave that door open.

**Comment by Tachi107** [comment]
Ok, I wrote GPLv3 to be consistent with the previous about dialog text. Do you prefer "GPLv3.0+" or "GPLv3.0-or-later"?

**Comment by FernandoS27** [comment]
> GPLv3.0-or-later

"GPLv3.0-or-later" is easier to understand.

**Comment by Tachi107** [comment]
Updated

image

**Review by FernandoS27 — APPROVED** [review]

File: dist/languages/zh_CN.ts [hunk 1]
```diff
@@ -27,14 +27,14 @@
...
         <translation>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu 是一个实验性的开源 Nintendo Switch 模拟器，以 GPLv2.0+ 授权。&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu 是一个实验性的开源 Nintendo Switch 模拟器，以 GPLv3.0++ 授权。&lt;/span&gt;&lt;/p&gt;
```
Comment (mzji) [inline dist/languages/zh_CN.ts:37]:
```Suggestion
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu 是一个实验性的开源 Nintendo Switch 模拟器，以 GPLv3.0+ 授权。&lt;/span&gt;&lt;/p&gt;
```

File: dist/languages/zh_TW.ts [hunk 2]
```diff
@@ -27,14 +27,14 @@
...
         <translation>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Ubuntu&apos;; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu是一個實驗性的開源Nintendo Switch模擬器，使用GPLv2.0+授權。&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu是一個實驗性的開源Nintendo Switch模擬器，使用GPLv3.0++授權。&lt;/span&gt;&lt;/p&gt;
```
Comment (mzji) [inline dist/languages/zh_TW.ts:37]:
```Suggestion
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:12pt;&quot;&gt;yuzu是一個實驗性的開源Nintendo Switch模擬器，使用GPLv3.0+授權。&lt;/span&gt;&lt;/p&gt;
```

**Pull request merged**